src/database.js

```diff
@@ -251,12 +257,6 @@ Database.prototype.batchTransaction = function(identifier) {
 Database.prototype.close = function(callback) {
   var key = this.id.split('/').pop();
   var leakError = null;
-  var leaks = this.pool_.getLeaks();
-
-  if (leaks.length) {
-    leakError = new Error(`${leaks.length} session leak(s) found.`);
-    leakError.messages = leaks;
-  }
 
   this.parent.databases_.delete(key);
   this.pool_.close().then(() => callback(leakError), callback);
```

Comment:
Removing this functionality would be a breaking change, can you please re-implement it?

Reply:
This functionality is not required as genericPool manages everything for us. I can implement something to "make it work" but that will be counter productive.

Reply:
I have implemented the leaks functionality.